en/sensor/pmw3901.md

```diff
@@ -0,0 +1,76 @@
+# PMW3901 Flow Sensor
+
+PMW3901 is an [optical flow](../sensor/optical_flow.md) asic that computes the flow internally and provides a difference in pixels between 
+each frame over a SPI interface. It is basically a tracking sensor similar to what you would find in a computer mouse, but adapted to work between 
+80 mm and infinity. The sensor can be read at a frequency up to 100 Hz.  
+
+![Bitcraze Flow Deck](../../assets/hardware/sensors/pmw3901/bitcraze-flow.jpg)
+
+The following is a video of an S500 frame with a Pixhawk 4 and the Bitcraze PMW3901 flow deck. GPS was fully disabled in this test to see the real performance of the flow module.
+
+!!!REPLACE WITH RIGHT VIDEO
+{% youtube %}
+https://youtu.be/0Jpq6DU_HVg
+{% endyoutube %}
+
+## Boards using PMW3901
+
+The following table shows some of the boards available on the market with the pmw3901 optical flow sensor. The number interfaces, the number of sensors, the input voltage and the size are shown. The board name contains the link to where to buy the specific board.
+
+Board | Interface | Flow | Range Finder | Gyro | Voltage (V) | Size (mm)
+--- | --- | --- | --- | --- | --- | ---
+<span id="Bitcraze"></span>[Bitcraze](https://www.bitcraze.io/flow-breakout) | SPI | X | X | - | 3 - 5 | AxB
+<span id="Tindie"></span>[Tindie](https://www.tindie.com/products/onehorse/pmw3901-optical-flow-sensor/) | SPI | X | - | - | 3 - 5 | AxB
+<span id="Hex"></span>[Hex](http://www.proficnc.com/all-products/185-pixhawk2-suite.html) | CAN | X | X | X | 3 - 5 | AxB
+<span id="Thone"></span>[Thone](https://www.seeedstudio.com/ThoneFlow-3901U-UART-Serial-Version-PMW3901-Optical-Flow-Sensor-p-4040.html) | UART | X | - | - | 3 - 5 | AxB
+
+## Specifications
+
+* TODO
+* TODO
+* TODO
+
+## Pixhawk Setup
+
+### Bitcraze
+
+The Bitcraze optical flow module exposes directly the SPI interface from the PMW3901 module. The board also has a distance sensor that can be wired via I2C. This distance sensor is the VL53L0x ToF sensor for STMicroelectronics. The range of the sensor is very limited (2 meters) and will be reduced when flying in the sunlight. We therefore highly recommend to use another external distance sensor. This board was mainly tested with the [Lidar Lite V3](../sensor/lidar_lite.md) on larger vehicles and with the [Lanbao CM8JL65](../sensor/cm8jl65_ir_distance_sensor.md) on smaller vehicles. 
```

Comment:
Not essential, but consider putting each sentence on a new line. Makes things slightly easier for translators.